knora-ontologies/knora-base.ttl

```diff
@@ -1195,11 +1195,9 @@
 
 :fps rdf:type owl:DatatypeProperty ;
 
-     rdfs:label "Frames per second"@en ;
-
      rdfs:subPropertyOf :valueHas ;
 
-     :subjectClassConstraint :MovingImageFileValue ;
+     :subjectClassConstraint :FileValue ;
```

Comment:
I think @subotic changed it. Because we no longer store the necessary moving image metadata such as width, height, duration and fps in the database. We get this information from the sidecar file from sipi.